packages/atomic/src/components/atomic-modal/atomic-modal.pcss

```diff
@@ -2,25 +2,6 @@
 
```

Comment:
the diff in this file is all messed up because I went from this layout:
```scss
some-component {
   // some generic style

  desktop {
    // some desktop style
  }

  mobile {
    // some mobile style
  }
}
```
to:
```scss
some-component {
  // some generic style
}

dialog {
  some-component {
    // some non-fullscreen style
  }
}

fullscreen {
  some-component {
    // some fullscreen style
  }
}
```

the only changes are that I added `p-6` to the backdrop and I changed `grid-template-columns` for the backdrop.